### DISCUSSION TITLE
[Question] Resize images as render in obsidian

### DISCUSSION THREAD

**Discussion by AlexShyXie** [mvdkwast/obsidian-copy-as-html#37] [Q&A]
Hello author, when I preview my notes in Obsidian, they are always fixed to the same size. 
However, when I export them to HTML, they are in their original size, which results in inconsistent image sizes. How can I set it so that the exported images are all the same size?

**In obsidian:**
![image](https://github.com/mvdkwast/obsidian-copy-as-html/assets/49368033/50a328d3-ad35-4f5f-bfa5-c4209ff3f1b4)

**After export in html:**
![image](https://github.com/mvdkwast/obsidian-copy-as-html/assets/49368033/c679a514-2ec6-4c75-a213-56c4f40f7734)

**Comment by mvdkwast** [comment]
Hello reporter ! I think you should be able to fix your image sizes using a custom stylesheet. 

I didn't test, but something like :
```css
img {
  width: 400px;
}
```

How the images are rendered may depend on where you paste the document unfortunately.

**Comment by mvdkwast** [comment]
Reporter didn't follow up, so assuming this is solved.
